### DISCUSSION TITLE
startsWithSearch() on Yajra V.8.00

### DISCUSSION THREAD

**Discussion by faisalachmadf** [yajra/laravel-datatables#2891] [Q&A] [answered]
Hello Everyone, 

i just wanna ask, Its Possible Using startsWithSearch() on Yajra V.8.0? 

cause i was found on the Doc, it worked on Yajra V.9++

Thanks...

**Comment by faisalachmadf** [comment]
@yajra could u give me some advice? 

thanks

**Comment by Arne1303** [comment, marked as answer]
The `startsWithSearch()` filter was added in [v9.5.0](https://github.com/yajra/laravel-datatables/blob/9.0/CHANGELOG.md#v950---2019-08-31) its not available in v8 you would need to upgrade to a newer version to use it
